### ISSUE TITLE
Keyboard nav only selects zoomed out book in Edge

### ISSUE THREAD

**Issue by BryceAG** [phetsims/friction#54]
The only object that can be highlighted and interacted with in Edge is the Chemistry Book, not even the zoomed in book will highlight. The reason could be a preference similar to #51 but I haven't found an option like that for Edge. 

Also the keyboard nav that does work is very delayed, taking about half a second before the book actually moves.

related to phetsims/QA#75

**Comment by jessegreenberg** [comment]
Also seeing very long delay in book movement in LoL harness in FF.

**Comment by jessegreenberg** [comment]
Also an issue in IE11.

**Comment by jessegreenberg** [comment]
I think this is caused by changes made for https://github.com/phetsims/scenery/issues/713.

**Comment by jessegreenberg** [comment]
I figured out this is due to  differences in the way IE11/Edge and the rest handle activeElement on blur.

For IE11/Edge, when blur happens, document.activeElement is the next focusable element.
For the rest, document.activeElement is `body`, until the next element receives the focus event.

In Display.js, we are counting on it being the element that was just blured to prevent.

**Comment by jessegreenberg** [comment]
Instead of using document.activeElement to blur active element, we can use Display.focusedNode because it will be consistent for all platforms.

**Comment by jessegreenberg** [comment]
I am no longer seeing delay in keyboard interaction in Edge. @BryceAG could you please add additional platform details so I can make sure that is no longer happening?

**Comment by jessegreenberg** [comment]
Should be fixed in the above commit,  verified in IE11 and Edge, still working in Chrome.

**Comment by zepumph** [comment]
This is cherry-picked to scenery's friction-1.4 branch

**Comment by EthanWJohnson** [comment]
This seems to be working fine in edge now.
